app.Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY --from=docker:dind /usr/local/bin/docker /usr/local/bin/
 
 # - "luarocks" – Lua package manager – https://luarocks.org/, https://packages.ubuntu.com/luarocks
 RUN apt update && \
-  apt install -y osm2pgsql osmium-tool postgresql-client-15 tzdata wget libpq-dev curl && \
+  apt install -y osm2pgsql osmium-tool postgresql-client-15 tzdata wget libpq-dev curl lua5.3 && \
```

Comment:
Habe das irgendwo im Wiki gelesen, dass es 5.3 sei. Lass uns das alles im Zuge des testing Tickets alles nochmal kontrollieren.

Reply:
[osm2pgsql docs](https://osm2pgsql.org/doc/manual.html#lua-script-performance):
> You can somewhat increase processing speed by using the LuaJIT compiler, but you have to compile osm2pgsql yourself, because pre-built osm2pgsql binaries usually don’t use it.

Unser osm2pgsql benutzt lua5.3.